integration_test/integration-test.go

```diff
@@ -0,0 +1,313 @@
+package main
+
+import (
+	"io/ioutil"
+	"fmt"
+	"net/http"
+	"net/url"
+	"time"
+	"strings"
+	"os"
+)
+
+/*****************************
+* INTEGRATION TEST FRAMEWORK *
+******************************/
+
+type Map = map[string]string
+
+/*
+* TestSuite struct
+*   keep track of info used by a test suite
+*   different proxy configurations corresponds
+*   to different test suites
+*/
+type TestSuite struct {
+	// url of the proxy to be tested
+	// the test client will send requests to this url
+	proxy_url string
```

Comment:
don't use underscores in Go names; struct field proxy_url should be proxyURL

Reply:
All the camel case renaming are in this [commit](https://github.com/dengyijia/WAF-wasm/pull/3/commits/f89b6330fe8f9137369359067fe79d8c7816e79b)